powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/GetErrorCodeListRequest.java

```diff
@@ -28,6 +29,7 @@
 @Data
 public class GetErrorCodeListRequest {
 
+    @Schema(description = "ISO 639 language code")
```

Comment:
🤔 Why not use the `format` attribute as elsewhere?
```suggestion
    @Schema(description = "Language code", type = "string", format="ISO 639")
```